pkg/api/types.go

```diff
@@ -1190,6 +1190,8 @@ type ServiceSpec struct {
 	// Type determines how the service will be exposed.  Valid options: ClusterIP, NodePort, LoadBalancer
 	Type ServiceType `json:"type,omitempty"`
 
+	LoadBalancerIP string `json:"loadBalancerIP, omitEmpty`
```

Comment:
It's an interesting case here.  We know that the status..ingress can be a name or IP and we split those fields on purpose.  In this case, is there anything equivalent in ELB-land @justinsb?  Can a user specify the value that eventually appears in the status..ingress.hostname?